src/core/InfoElementCfg.h

```diff
@@ -1,6 +1,7 @@
 #ifndef INFOELEMENTCFG_H_
 #define INFOELEMENTCFG_H_
 
+#include <utility>
```

Comment:
Apparently not, just tried to build without it on Debian 9, can't remember now why it was added
Removed